.circleci/config.yml

```diff
@@ -22,30 +25,37 @@ jobs:
       - restore_cache:
           key: deps-{{ .Branch }}-{{ checksum "ci/pyuvdata_tests.yml" }}
       - run:
-          name: Install miniconda
-          command: |
-            wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O $HOME/miniconda.sh;
-            bash $HOME/miniconda.sh -b -p $HOME/miniconda
+          name: Instal Micromamba
+          command: wget -qO- https://micro.mamba.pm/api/micromamba/linux-64/latest | sudo tar -C / -xvj bin/micromamba
       - run:
           name: configure conda environment
           command: |
-            export PATH="$HOME/miniconda/bin:$PATH"
-            ./ci/install_conda.sh
+            micromamba info
+            # need these to add gxx and gcc to build novas and cython
+            micromamba create --name=${ENV_NAME}  python=$PYTHON gxx gcc -f ci/${ENV_NAME}.yml -yq
```

Comment:
Does this mean we aren't using "install_conda.sh" in CircleCI any more?

Reply:
That's correct we're not. I can put some of this back in there if we'd like but ended up pulling it out while trying to debug.